v3/pkg/application/systemtray_windows.go

```diff
@@ -117,6 +122,22 @@ func (s *windowsSystemTray) bounds() (*Rect, error) {
 	}, nil
 }
 
+func (s *windowsSystemTray) iconIsInTrayBounds() (bool, error) {
```

Comment:
I'd recommend just returning bool here so you can just use the function call instead of making a local variable with the result named the same thing.

Reply:
We do need to handle that error though.